ansible/roles/common_installs/tasks/main.yml

```diff
@@ -33,6 +33,11 @@
   sudo: yes
   with_items:
     - less@1.3.1
+    - n@1.3.0
```

Comment:
haha yup: http://davidwalsh.name/upgrade-nodejs
i guess it's so presumptuous that no one does it and thus isn't used